packages/eslint-plugin-udemy/rules/no-settimeout/README.md

```diff
@@ -1,6 +1,6 @@
-# No Non-Zero setTimeout
+# No setTimeout
 
-`settimeout/no-nonzero` rule checks for any uses of `window.setTimeout` with a non-zero delay value.
+`udemy/no-settimeout` rule checks for any uses of `window.setTimeout.
```

Comment:
It seems like you're missing a backtick.

Reply:
🤦Fixed.